src/components/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ name }) => (
+  <button type="button">
+    {name}
+  </button>
+);
+
+Button.propTypes = {
+  name: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: '0',
+};
```

Comment:
we may not need this. we want every button to have a name, so we could attach required to the propTyes of name in addition to enforcing its a string.